core/services/pipeline/orm_test.go

```diff
@@ -0,0 +1,150 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	gormv1 "github.com/jinzhu/gorm"
+	p "github.com/smartcontractkit/chainlink/core/services/postgres"
+	"github.com/smartcontractkit/chainlink/core/store/migrations"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/guregu/null.v4"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func TestClaimTaskScale(t *testing.T) {
```

Comment:
I think we could leave this in but just t.Skip it?

Reply:
IIRC golang actually has a performance testing mode? Can it fall under that?

Reply:
Sorry its called benchmark mode

Reply:
oh I think you just mean the benchmarking? yeah I was thinking about doing that but wanted to still use the testify stuff. Seems easy enough to just write my own requireNoError function though so ill go ahead and convert it to a benchmark